lib/console_manager.rb

```diff
@@ -110,6 +128,18 @@ def list_all_music
   # list of games
 
   # list all genres
+  def list_all_genres
+    genres = @music_manager.music_list.map(&:genre).compact
```

Comment:
Here, you can simply use 
```
genres = @genres_manager.genres_list
```
I used this approach earlier because I did not have a LabelsManager class at the time.